scripts/provision/provision_cplane.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
This is the provisioning bash script specific to a "control plane" (cplane) nodes, and run during the Vagrant "up" state.